console/console-init/ui/src/modules/address-space/components/EditAddressSpace/EditAddressSpace.tsx

```diff
@@ -3,12 +3,7 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import React, { useState } from "react";
-import { useQuery } from "@apollo/react-hooks";
-import {
-  RETURN_ADDRESS_SPACE_PLANS,
-  RETURN_FILTERED_AUTHENTICATION_SERVICES
-} from "graphql-module/queries";
+import React from "react";
```

Comment:
same here. Any specific reason to just import React instead of * as React statment?

Reply:
don't need to use * as i explained in other file.